scripts/jenkins/Jenkinsfile

```diff
@@ -936,6 +945,7 @@ def buildAndTest(testName, image, extraCmakeFlags = [:],
 
         if(tests) {
           artifacts.add("\"${buildDir}\"/Testing/*/*.xml")
+          artifacts.add("\"${buildDir}\"/Testing/Temporary/MemoryChecker.*.log")
```

Comment:
@petermax2 I use this to add the valgrind output to the artifacts. Can not be done on master (uses too much space), but can be enabled in a PR at any time to have non-scrambled valgrind output.